action.yml

```diff
@@ -2,52 +2,51 @@ name: Update Deployment Board
+  token:
+    description: A token with permissions to create and update issues
     required: true
   environment:
-    description: 'The environment the branch, tag or SHA was deployed to'
+    description: The environment the branch, tag or SHA was deployed to, i.e. [DEV|QA|STAGE|DEMO|UAT|PROD]
     required: true
-  board-number:
-    description: 'The number of the project board that will be updated.  Found by using the number in the board url.'
+  project-slug:
+    description: The repo the deployment will be created in, i.e. [org|owner]/[repo name]
     required: true
   ref:
-    description: 'The branch, tag or SHA that was deployed'
+    description: The branch, tag or SHA that was deployed
     required: true
-  ref-type:
-    description: 'The type of ref that was deployed  [branch|tag|sha].  If not provided the action will use some regex patterns to identify the type.'
-    required: false
-  deployable-type:
-    description: 'The type of artifact that was deployed like MFE, BFF, API, SVC, DB, etc.  Useful when a release with one tag contains multiple deployable artifacts that are deployed separately.'
-    required: false
-  deploy-status:
-    description: 'The status of the deployment [success|failure|cancelled|skipped]'
+  deployment-status:
+    description: The status of the deployment [error|failure|success]
     required: true
-  deploy-label:
-    description: 'The optional label of the deployment [deleted|destroyed|<your-custom-label>]'
+  deployment-message:
+    description: Any message to be delivered along side of the status
     required: false
-  enable-deployment-slot-tracking:
-    description: 'Enable App Service deployment slot tracking on deployment board? [true|false]'
-    required: false
-    default: 'false'
-  slot-swapped-with-production-slot:
-    description: 'Did this deployment swap slots with production slot? Set to true for swapping slot immediately with production slot. [true|false]'
+  deployment-auto-inactivate:
+    description: |
+      Automatically mark prior environment deployments inactive, only use 'true' for environments that do not
+      have multiple instances, expects 'true' or 'false', defaults to 'false'
     required: false
     default: 'false'
-  target-slot:
-    description: 'The target slot that was deployed to [production|blue|yellow|canary|red|loadtest|predeploy|your-custom-slot]'
-    required: false
-    default: 'production'
-  source-slot:
-    description: 'The source slot that was deployed to [production|blue|yellow|canary|red|loadtest|predeploy|your-custom-slot]'
-    required: false
-    default: 'production'
-  timezone:
-    description: 'IANA time zone name (e.g. America/Denver) to display dates in.  If timezone is not provided, dates will be shown in UTC'
-    required: false
+  entities:
+    description: The entities that are affected by the deployment, i.e. ["proj-app", "proj-infrastruction", "proj-db"]
+    required: true
```

Comment:
There should only be one entity per deployment object. If there's a set of things that deploy together, they should be modelled as one entity. If they _can_ be deployed separately, then they should have separate deployment objects.

We also need to explain that the value of this input needs to match the entity id of the Backstage (TechHub) catalog.